spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -139,21 +144,14 @@ function setup() {
 }
 ```
 
-Once the `setup` function has been called, channels can be created through the IBC routing module between instances of the ibc account module on separate chains.
-
-An administrator (with the permissions to create connections & channels on the host state machine) is responsible for setting up connections to other state machines & creating channels
-to other instances of this module (or another module supporting this interface) on other chains. This specification defines packet handling semantics only, and defines them in such a fashion
-that the module itself doesn't need to worry about what connections or channels might or might not exist at any point in time.
-
-### Routing module callbacks
+Once the `setup` function has been called, channels can be created via the IBC routing module.
 
 ### Channel lifecycle management
 
-Both machines `A` and `B` accept new channels from any module on another machine, if and only if:
+An interchain account module will accept new channels from any module on another machine, if and only if:
 
-- The other module is bound to the "ibcaccount" port.
-- The channel being created is ordered.
-- The version string is "ics27-1".
+- The channel being created is ordered
+- The version string is "ics27-1"
 
 ```typescript
 function onChanOpenInit(
```

Comment:
We should be erroring here and on all handshake messages if `activeChannel` already set

Reply:
Updated this